config/config.exs

```diff
@@ -21,6 +21,9 @@ use Mix.Config
 #     config :logger, level: :info
 #
 
+config :gtfs_realtime_viz,
```

Comment:
I think using `config` for libraries is now considered an anti-pattern. I started out this way configuring the routes and the `max_archive` that way, but after reading more on it, I think that's not the way to go. For instance, see [this discussion](https://elixirforum.com/t/mix-config-evolutions/4423/9) where Sasa Juric and Jose Valim and others discuss it and recommend, if possible, configuring your library through function arguments. Otherwise it gets tricky setting the configuration with regard to compilation and releases.

So I think it would be better to put the `timezone` in the `opts` parameter of `visualize/2` and `visualize_diff/3`.

Reply:
Hm, just realized `opts` is actually the routes that we want to visualize (so might be a bit of a confusing parameter name). Maybe rename that parameter as `route_config` or something, and then add another `opts` parameter to store stuff like `timezone`?

Reply:
good point